src/OpenSage.Game/Gui/DebugUi/DebugDrawable.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Numerics;
+using OpenSage.Graphics.Cameras;
+using OpenSage.Mathematics;
+
+namespace OpenSage.Gui.DebugUi
```

Comment:
The namespace should be DebugUI (if I understand the current .NET. Among conventions correctly). I haven't always followed them myself, but we might as well use them for new code.